package.json

```diff
@@ -70,6 +70,7 @@
     "drive-bucket-sync": "node scripts/drive-sync.js",
     "update-media-urls": "node scripts/update-urls.js",
     "save-captions": "node scripts/save-captions.js",
+    "save-bucket-captions": "node scripts/save-bucket-captoions.js",
```

Comment:
Filename typo: `save-bucket-captoions.js` -> `save-bucket-captions.js`

Reply:
@calym I checked the Drive and yes, those captions are no longer needed (https://d.pr/i/CT7WvI). I'll remember to ask the contractor to fill in the missing caption paths as part of the next milestone.